.github/workflows/dev_pr.yml

```diff
@@ -1,51 +0,0 @@
-# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
The labeler job isn't going to be useful if it just puts the object_store label on every PR so I removed it